src/main/java/seedu/address/logic/commands/UntagCommand.java

```diff
@@ -0,0 +1,115 @@
+package seedu.address.logic.commands;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Removes a tag associated with an existing person in the address book.
+ */
+public class UntagCommand extends Command {
+
+    public static final String COMMAND_WORD = "untag";
+    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid.";
+    public static final String MESSAGE_TAG_NOT_FOUND = "Some tags were not found in the person's tag list.";
+    public static final String MESSAGE_REMOVE_TAG_SUCCESS = "Removed tag(s) %1$s from %2$s.";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Removes one or multiple tags from the person identified "
+            + "by the index number used in the last person listing.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "[TAG]... (can specify multiple tags)\n"
+            + "Example: " + COMMAND_WORD + " 1 "
+            + "florist photographer.";
+
+    private final Index index;
+    private final List<Tag> tagsToRemove;
```

Comment:
Type of tagsToRemove could follow the type in Person.java, HashSet for consistency?

Reply:
True, ill make the change, thanks